tests/baselines/reference/errorOnInitializerInObjectType.errors.txt

```diff
@@ -0,0 +1,17 @@
+tests/cases/compiler/errorOnInitializerInObjectType.ts(2,17): error TS1246: An object type property cannot have an initializer.
+tests/cases/compiler/errorOnInitializerInObjectType.ts(6,17): error TS1246: An object type property cannot have an initializer.
+
+
+==== tests/cases/compiler/errorOnInitializerInObjectType.ts (2 errors) ====
+    interface Foo {
+        bar: number = 5;

```

Comment:
We should just allow initializers.  This is a pretty reasonable thing for a user to type, so the tree shoudl just be flexible enough to represent it.  We can hten have a simple grammar check in the type checker that reports that thsi isn't legal.